jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,17 @@
+lib = library(identifier: 'jenkins@1.5.3', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
```

Comment:
Need manually change in the jenkins workflow settings.

Reply:
Correct, this is why I pinged you and @gaiksaya

Reply:
Hey @reta I am out this week a bit. I can take this up next week if it is not urgent. Let me know. 
Thanks!

Reply:
@gaiksaya could you help to move it forward please? thank you

Reply:
Working on adding the credentials. Overall the changes look good to me